docs/en/docs/lakehouse/multi-catalog/jdbc.md

```diff
@@ -146,6 +146,27 @@ Once connected, Doris will ingest metadata of databases and tables from the exte
    | Database | Schema    |
    | Table    | Table     |
 
+<version since="1.2.2">
```

Comment:
move the <version> to wrap the `Jdbc Catalog also support to connect another Doris database:`, not wrap the title

Reply:
and use "since='dev'"

Reply:
done